projects/telemetry/inc/util.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <stdbool.h>
+#include <stdint.h>
+
+// Computes the checksum for a given NMEA message.
+//
+// Starts at the second characer of `message`,
+// and continues until the `*` character.
+//
+// `out` must be a pointer to at least 3 `char`s.
+void evm_gps_compute_checksum(char *message, char *out);
```

Comment:
Please fix your naming scheme to match the standard and make sure that your inputs take length parameters as well in case they're malformed.